CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# HEAD
+
+## Features
+* **css:** Migrates the sass `@import` with `@use` and `@forward` (#755).
+
```

Comment:
We should write up migration steps here for what has to happen on the consumer side when updating.